realm/realm-library/src/objectServer/java/io/realm/SyncSession.java

```diff
@@ -53,6 +59,8 @@
 public class SyncSession {
     private final static ScheduledThreadPoolExecutor REFRESH_TOKENS_EXECUTOR = new ScheduledThreadPoolExecutor(1);
     private final static long REFRESH_MARGIN_DELAY = TimeUnit.SECONDS.toMillis(10);
+    private final static int DIRECTION_DOWNLOAD = 1;
+    private final static int DIRECTION_UPLOAD = 2;
```

Comment:
Have any good ideas for what that class might be. I hadn't which was why I settled on this.